servicetalk-tcp-netty-internal/src/main/java/io/servicetalk/tcp/netty/internal/TcpServerConfig.java

```diff
@@ -52,6 +53,7 @@ public final class TcpServerConfig extends AbstractTcpConfig<ServerSslConfig> {
     private TransportObserver transportObserver = NoopTransportObserver.INSTANCE;
     @Nullable
     private Map<String, ServerSslConfig> sniConfig;
+    private SslListenMode sslListenMode;
```

Comment:
Does this need a default value, presumably `SSL_REQUIRED`?

Reply:
👍